fetch-api/fetch-post.html

```diff
@@ -0,0 +1,119 @@
+limitations under the License.
+-->
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="description" content="Sample illustrating the use of Fetch API.">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Fetch API POST Sample</title>
+
+    <!-- Add to homescreen for Chrome on Android -->
+    <meta name="mobile-web-app-capable" content="yes">
+    <link rel="icon" sizes="192x192" href="../images/touch/chrome-touch-icon-192x192.png">
+
+    <!-- Add to homescreen for Safari on iOS -->
+    <meta name="apple-mobile-web-app-title" content="Fetch API Reddit Sample">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <link rel="apple-touch-icon-precomposed" href="../images/apple-touch-icon-precomposed.png">
+
+    <!-- Tile icon for Win8 (144x144 + tile color) -->
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+</head>
+
+<body>
+<h1>Fetch API POST Sample</h1>
+
+<p>Available in <a href="http://www.chromestatus.com/feature/6730533392351232">Chrome 42+</a></p>
+
+<div class="output">
+    <p id="log"></p>
+</div>
+
+<!-- // [START code-block] -->
+<script>
+    "use strict";
+
+    var logEl =  document.querySelector('#log');
+
+    function log() {
+        logEl.innerHTML += '<p>' + Array.prototype.join.call(arguments, '') + '<p>';
+    }
+
+    function serialize(data) {
+        if(data) {
+            return Object.keys(data).map(function (key) {
+                return encodeURIComponent(key) + '=' + encodeURIComponent(data[key])
+            }).join('&');
+        }
+    }
+
+    /**
+     *
+     * @param url URL for validation
+     * @param json JSON to validate
+     * @note adapted from http://blog.gospodarets.com/fetch_in_action/
+     */
+    function validate(url, json) {
+        json = typeof json === 'object' ? JSON.stringify(json) : json;
+        return fetch(url, {
+            method: 'post',
+            headers: {
+                "Content-type": "application/x-www-form-urlencoded; charset=UTF-8"
+            },
+            body: serialize({json: json})
+        }).then(function (response) {
+            return response.json();
+        }).then(function (data) {
+            if (data.validate) {
+                log('String: "' + json + '" is valid JSON');
+            } else {
+                log('String: "' + json + '" is not valid JSON');
+                log(data.error);
+            }
+        }).catch(function (error) {
+            log(error.message);
+        });
+    }
+
+    var jsonValidator = 'http://validate.jsontest.com/';
```

Comment:
You'll get a mixed content failure when the sample page is accessed via HTTPS.

`Mixed Content: The page at 'https://rawgit.com/GoogleChrome/samples/fetch/fetch-api/fetch-post.html' was loaded over HTTPS, but requested an insecure resource 'http://validate.jsontest.com/'. This request has been blocked; the content must be served over HTTPS.`

And it looks like https://validate.jsontest.com/ isn't a thing.